runtime/compiler/codegen/J9CodeGenerator.hpp

```diff
@@ -442,15 +442,25 @@ void addMonClass(TR::Node* monNode, TR_OpaqueClassBlock* clazz);
    void setSupportsInlineStringCaseConversion() { _j9Flags.set(SupportsInlineStringCaseConversion);}
 
    /** \brief
-    *    Determines whether the code generator supports inlining of java/lang/String.indexOf()
+    *    Determines whether the code generator supports inlining of java/lang/String.indexOf(int)
     */
    bool getSupportsInlineStringIndexOf() { return _j9Flags.testAny(SupportsInlineStringIndexOf);}
 
    /** \brief
-    *    The code generator supports inlining of java/lang/String.indexOf()
+    *    The code generator supports inlining of java/lang/String.indexOf(int)
     */
    void setSupportsInlineStringIndexOf() { _j9Flags.set(SupportsInlineStringIndexOf);}
 
+   /** \brief
+    *    Determines whether the code generator supports inlining of java/lang/String.indexOf(String)
+    */
+   bool getSupportsInlineStringIndexOfString() { return _j9Flags.testAny(SupportsInlineStringIndexOfString);}
+
+   /** \brief
+    *    The code generator supports inlining of java/lang/String.indexOf()
```

Comment:
This should be same as getter (`String.indexOf(String)`)

Reply:
Fixed.